examples/Complex-Sounding-Plot.py

```diff
@@ -0,0 +1,318 @@
+# Copyright (c) 2015,2016,2017 MetPy Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
+=================
+Sounding Plot with Complex Layout
+=================
+
+This example combines simple MetPy plotting functionality, `metpy.calc`
+computation functionality, and a few basic tricks to create an 
```

Comment:
Part of our linting (format) requirements is that there shouldn't be any extra spaces at the end of a line. The newline acts like a space, so you'll just have to clean up a few of those, which are all detailed by the GitHub Actions/Flake 8 errors.

Reply:
On it, thank you!